src/src/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -136,28 +148,68 @@ public boolean canUseLocalCache() {
 
         return false;
     }
-
+        
     /**
-     * App needs to give permission to AccountManager to use broker.
+     * To verify if App gives permissions to AccountManager to use broker.
+     * 
+     * @throws DeveloperAuthenticationException
+     * @return boolean If the GET_ACCOUNTS, MANAGE_ACCOUNTS, USE_CREDENTIALS permissions are granted in the Manifest.xml
      */
     private boolean verifyManifestPermissions() {
```

Comment:
If there is a apparent "throw exception" line in the code, then there is no need to add the "throws" behind the method name for this exception.